tests/test_aamp_mmotifs.py

```diff
@@ -0,0 +1,351 @@
+import numpy as np
+import numpy.testing as npt
+import naive
+
+from stumpy.aamp_mmotifs import aamp_mmotifs
+from stumpy import config
+
+
+# These are tests for non-normalized multidimensional motif discovery
+
+
+def test_motifs_multidimensional_one_motif_all_dimensions():
+    # Find the two-dimensional motif pair
+
+    # Arrange
+    motif_distances_expected = np.array([[0.0, 2.87778559]])
+    motif_indices_expected = np.array([[0, 5]])
+    motif_subspaces_expected = [np.array([2, 1])]
+    motif_mdls_expected = [np.array([244.0, 260.67970001, 279.86313714, 281.35940001])]
+
+    # Act
+    T = np.array(
```

Comment:
If this array is the same in all tests, can you please follow other tests and declare the array once at the beginning like (see `tests/test_stumpy.py`: 

```
test_data = [np.array([9, 8100, -60, 7], dtype=np.float64)]
```
and then you can pass it to your test with:

```
@pytest.mark.parametrize("T", test_data)
def test_aamp_mmotifs_2D_motif_pair(T):
    .
    .
    .
    P, I = naive.maamp(T, m, excl_zone)
```

Reply:
>If this array is the same in all tests, can you please follow other tests and declare the array once at the beginning like (see tests/test_stumpy.py:

The array has to be the same every time we are searching for the first motif. But once we discover the top two motifs we get the following: 
```
motif_mdls_ref = [
        np.array([244.0, 260.67970001, 279.86313714, 281.35940001]),
        np.array([254.33985, 260.67970001, 279.86313714, 291.20703549]),
    ]
```

Reply:
I'm not sure I understand what you mean. I'm just asking whether `T` is the same in all of the unit tests? If so, then please define `T` once at the top of the test file as `test_data` and then pass `test_data` into each of your tests via:
```
@pytest.mark.parametrize("T", test_data)
def test_aamp_mmotifs(T):
```

Reply:
Ohh, I'm sorry! 
Sometimes it's a bit difficult to tell which line you mean 😄

Reply:
@SaVoAMP In case it matters, in Github, I usually assign the comment to the precise line of code (or start of the code chunk) that I am referring to.  Github usually displays the comment directly under the "line of interest" and, in this case, I chose the starting line of:

```
T = np.array(
```

Although, based on what is displayed, I can see why you thought I was referring to the `motif_mdls_expected =` line now.